src/viam/ur/module/ur_arm.cpp

```diff
@@ -1188,7 +1187,7 @@ URArm::UrDriverStatus URArm::read_joint_keep_alive_inner_(bool log) {
 
     } else {
         // the arm is in a normal state.
-        if (current_state_->estop.load()) {
+        if (current_state_->estop.load() && !current_state_->local_disconnect.load()) {
```

Comment:
whoops

Reply:
Basically, don't attempt to recover from estop if we are already in local mode?

Reply:
yup. We don't really want to be interacting with the machine while in local mode. So should probobably switch the order of these to highlight that context and short-circuit when possible